### ISSUE TITLE
Asus STRIX RX580

### ISSUE THREAD

**Issue by michlinux** [twifty/aura-gpu#9]
My Asus STRIX RX580 isn't being detected in OpenRGB in Linux.
Sometime over a year ago my Asus STRIX RX580 was detected and worked in OpenRGB in Linux, im not sure when this stopped being the case. However it still works in Windows 10 perfectly. I know there is a bit of an issue with i2c being exposed by the amdgpu module, but making this report as it used to work and this is one of the supported cards. I have compiled the aura-gpu driver and it seams to be loading and working correctly, at least it is detecting the card, except it seams to be missing some values in the i2cdetect -y 10. Thank you and please let me know if you need anymore info. Also included OpenRGB Log(loglevel 6) and dmesg log.
```
[XXXXX@localhost ~]$ i2cdetect -l
i2c-0   i2c             AMDGPU DM i2c hw bus 0                  I2C adapter
i2c-1   i2c             AMDGPU DM i2c hw bus 1                  I2C adapter
i2c-2   i2c             AMDGPU DM i2c hw bus 2                  I2C adapter
i2c-3   i2c             AMDGPU DM i2c hw bus 3                  I2C adapter
i2c-4   i2c             AMDGPU DM i2c hw bus 4                  I2C adapter
i2c-5   i2c             AMDGPU DM aux hw bus 0                  I2C adapter
i2c-6   i2c             AMDGPU DM aux hw bus 1                  I2C adapter
i2c-7   smbus           SMBus PIIX4 adapter port 0 at 0b00      SMBus adapter
i2c-8   smbus           SMBus PIIX4 adapter port 2 at 0b00      SMBus adapter
i2c-9   smbus           SMBus PIIX4 adapter port 1 at 0b20      SMBus adapter
i2c-10  i2c             AURA GPU adapter                        I2C adapter
[XXXXX@localhost ~]$ i2cdetect -y 10
     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00:                         08 -- -- -- -- -- -- -- 
10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
20: -- -- -- -- -- -- -- -- -- 29 -- -- -- -- -- -- 
30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
60: -- -- -- -- -- -- -- -- 68 -- -- -- -- -- -- --
[XXXXX@localhost ~]$ uname -r
5.13.16-200.fc34.x86_64
[XXXXX@localhost aura-gpu]$ uname -r
5.14.16-301.fc35.x86_64
[XXXXX@localhost OpenRGB]$ lspci -d 1002: -nnvm | head -6 | tail -n 4 && lspci -d 10DE: -nnvm | head -6 | tail -n 4 
Vendor: Advanced Micro Devices, Inc. [AMD/ATI] [1002]
Device: Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] [67df]
SVendor:        ASUSTeK Computer Inc. [1043]
SDevice:        Device [0517]
```
[dmesg.log](https://github.com/twifty/aura-gpu/files/7492582/dmesg.log)
[OpenRGB.log](https://github.com/twifty/aura-gpu/files/7492584/OpenRGB.log)

**Comment by michlinux** [comment]
I decided to try an older version of OpenRGB and it works so clearly not the module at fault.

**Comment by crashniels** [comment]
I fixed that via https://github.com/twifty/aura-gpu/pull/8

**Comment by michlinux** [comment]
> I fixed that via #8

Thank you very much, didnt notice the pull request - my bad..
it does indeed fix my problem.
